server/services/repositories/ConversationRepository.js

```diff
@@ -12,6 +12,20 @@ module.exports = class ConversationRepository extends BaseRepository {
         this.userRepository = userRepository;
     }
 
+    async create(model) {
+
+        try {
+            let schemaToSave = this.toSchema(model);
+            let savedSchema = await schemaToSave.save();
+            savedSchema = await savedSchema.populate('users').execPopulate();
+            return this.fromSchema(savedSchema);
+        }
+        catch (err) {
+            return Promise.reject(err);
+        }
+
+    }
```

Comment:
I think this change updated what's returned from the API. When creating a new conversation, the user's name isn't shown. You may need to update how you access the data on the client side when creating a new conversation.

![image](https://user-images.githubusercontent.com/30205106/117050521-6eb76b00-ace3-11eb-88e8-0787c544d9f3.png)

Reply:
This was an existing bug. This change fixes it

Reply:
Whoops, I didn't pull down the latest changes. Thanks for letting me know.